.github/workflows/release.yml

```diff
@@ -0,0 +1,28 @@
+name: release
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  syndicate:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v3
+    - name: Install Task
+      run: |
+        curl -sL https://taskfile.dev/install.sh | sh
+        sudo mv ./bin/task /usr/local/bin
+    - name: Setup Go
+      uses: actions/setup-go@v4
+      with:
+        go-version: 1.21
+    - name: Sync with GoDocs using Taskfile
+      run: task godocs
+      env:
+        GIT_DOMAIN: ${{ secrets.GIT_DOMAIN }}
+        REPO_OWNER: ${{ secrets.REPO_OWNER }}
+        REPO_NAME: ${{ secrets.REPO_NAME }}
```

Comment:
I would try to get those values rather from github action context (like event or something) then force us to set those values manually.